test/test_qgis2web_dialog.py

```diff
@@ -23,6 +23,7 @@
 from qgis.core import QgsVectorLayer, QgsMapLayerRegistry, QgsCoordinateReferenceSystem
 from PyQt4 import QtCore, QtTest
 from PyQt4.QtCore import *
+from osgeo import gdal
```

Comment:
It's tricky. I think a better approach here might be to have a set of acceptable control files, and if the test matches either then it passes. FYI the difference is that on travis coordinates like 18.00 will be truncated to 18.0, but on newer gdal they will get fully truncated to just '18'